lib/ui/painting/paint.cc

```diff
@@ -47,6 +48,11 @@ constexpr uint32_t kBlendModeDefault =
 // default SkPaintDefaults_MiterLimit in Skia (which is not in a public header).
 constexpr double kStrokeMiterLimitDefault = 4.0;
 
+// A color matrix which inverts colors.
+constexpr SkScalar invert_colors[20] = {-1.0, 0,   0,   1.0, 0,   0,    -1.0,
```

Comment:
Format this in a nice 4x5 matrix? The current 3x* matrix is hard to read...

Reply:
This is what clang-format gave me :(

Reply:
Yeah, that would be helpful